README.md

```diff
@@ -1,18 +1,71 @@
 # `warehouse.ai-ui`
 
-UI for warehouse.ai
+[![Version npm](https://img.shields.io/npm/v/warehouse.ai-ui.svg?style=flat-square)](https://www.npmjs.com/package/warehouse.ai-ui)
+[![License](https://img.shields.io/npm/l/warehouse.ai-ui.svg?style=flat-square)](https://github.com/godaddy/warehouse.ai-ui/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/warehouse.ai-ui.svg?style=flat-square)](https://npmcharts.com/compare/warehouse.ai-ui?minimal=true)
+[![Build Status](https://travis-ci.org/godaddy/warehouse.ai-ui.svg?branch=master)](https://travis-ci.org/godaddy/warehouse.ai-ui)
+[![Dependencies](https://img.shields.io/david/godaddy/warehouse.ai-ui.svg?style=flat-square)](https://github.com/godaddy/warehouse.ai-ui/blob/master/package.json)
+
+UI for [Warehouse.ai]. Visual representation of your builds and status events.
+
+## Install
+
+```
+git clone git@github.com:godaddy/warehouse.ai-ui.git
+cd warehouse.ai-ui && npm install
+```
 
 ## Usage
 
+The module provides a [`bin/server`](./bin/server) script that starts the UI.
+Run the UI with `npm`.
+
+```bash
+npm start
+```
+
+The client-side bundle already comes packaged with this `npm` module, but if
+you need to re-build the assets, for whatever reason, you can do so as
+follows:
+
+```sh
+npm run build
+```
+
+## API
+
+### GraphQL API
+
+Warehouse UI is backed by a GraphQL API. This API transforms the data from the
+existing warehouse API to a model that is specific to this UI. The GraphQL
+models are based on [warehouse-models] and [warehouse.ai-status-models].
```

Comment:
`warehouse.ai-status-models` isn't defined. Should this be `warehouse-status-models`?